### ISSUE TITLE
A question about the loss of model

### ISSUE THREAD

**Issue by lizzyYL** [argman/EAST#127]
【code:】
model.py

def loss(y_true_cls, y_pred_cls,
         y_true_geo, y_pred_geo,
         training_mask):
    ……
    ……
    L_g = L_AABB + 20 * L_theta

    return tf.reduce_mean(L_g * y_true_cls * training_mask) + classification_loss

【question:】
You return the loss (…… L_g * y_true_cls * ……）.  Is y_true_cls a 0-1 matrix ,and 0 is background as well as 1 is text boxes? Why did you multiply y_true_cls here? 
May this make the loss is only related to the text area , nothing about the backgroud information? 
Respectfully waiting for a reply .Thank you:)

**Comment by UpCoder** [comment]
I guess it same with the Faster RCNN which don't care the negative anchors when calculate the smooth L1 loss. We use the classification_loss to ensure the background  is classified to background. And If we remove the y_true_cls, I think the loss of this part is too large to converge.
The above is just my understanding, 
Respectfully waiting for a replay. Thank you.

**Comment by argman** [comment]
Doing regression on non-text area is meaningless

**Comment by UpCoder** [comment]
I find the classification loss isn't balanced cross-entropy, it is dice_loss that isn't same with the paper. So why you use the dice_loss instead of balanced cross-entropy loss?

Respectfully waiting for a reply .Thank you:)
